libs/shared/components/src/auth/UserMenu/userProfile.graphql.ts

```diff
@@ -8,3 +8,11 @@ export const USER_PROFILE = gql`
     }
   }
 `
+
+export const UPDATE_USER_PROFILE = gql`
+  mutation updateProfile($input: UpdateUserProfileInput!) {
```

Comment:
Shouldn't this be called `updateUserProfile`?

Reply:
Makes sense, updated!